lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -501,15 +505,15 @@ EOF
     my $detect_signature = script_output("cat $args{resolvconf} | grep \"#Modified by parallel_guest_migration_base.pm module\"", proceed_on_failure => 1);
     my $detect_name_server = script_output("cat $args{resolvconf} | grep \"nameserver $args{resolvip}\"", proceed_on_failure => 1);
     my $detect_domain_name = script_output("cat $args{resolvconf} | grep \"$args{domainname}\"", proceed_on_failure => 1);
-    $ret |= assert_script_run("cp $args{resolvconf} /etc/resolv_backup.conf") if ($detect_signature eq '');
-    $ret |= assert_script_run("awk -v dnsvar=$args{resolvip} \'done != 1 && /^nameserver.*\$/ { print \"nameserver \"dnsvar\"\"; done=1 } 1\' $args{resolvconf} > $args{resolvconf}.tmp") if ($detect_name_server eq '');
+    $ret |= script_run("cp $args{resolvconf} /etc/resolv_backup.conf") if ($detect_signature eq '');
```

Comment:
For what cases do these replacement of assert_script_run to script_run needed? What's the plan for essential fix of relevant network connectivity?

Reply:
Oh, that seems a little weird... You mean we lose the check of assert, to remove warning for "uninitialized"? Does not seem quite reasonable to me, to be honest.

Reply:
@waynechen55 said it is ok to make these changes.  @alice-suse What do you think of it?

Reply:
Ok, to remove `$ret |= ` from all `$ret |= assert_script_run()` in this function.

Reply:
I will have a look, maybe let it die if return value is not zero.